decidim/decidim
# Apply flash styles to Announcements

#### :tophat: What? Why?

As I mentioned when I introduced the new flashes in the admin panel (#11665): 

> NOTE: it's out of the scope of the current PR to actually change the announcements/callouts that are mixed in some forms. The idea is to make a new PR after this one is merged with those changes, to keep them small.

So, this is the PR missing, with the change of the announcements to use the new design. 

Apart from that, I tuned a bit the CSS of the flashes, as I found them too bold. 

Its worth noticing two particular cases where we used links in the texts, so on those cases I preferred to hard-code the styling, as the alternative was relaxing the security in the AnnouncementCell clean_body method and I didn't like that


#### :pushpin: Related Issues
 
- Related to #11665

#### Testing

1. Go to a place with announcements, for instance:
- Organization colors: http://localhost:3000/admin/organization/appearance/edit
- Conference invitations: http://localhost:3000/admin/conferences/highway-fruit/conference_invites/new 
- Delete my account: http://localhost:3000/account/delete 
2. See that the styles are new  

### :camera: Screenshots

![Screenshot of conference's invitations](https://github.com/decidim/decidim/assets/717367/02898e46-d60e-44fa-b003-62d5ed2fdc80)


:hearts: Thank you!
